registrations/tests/test_signup_put.py

```diff
@@ -35,34 +39,52 @@ def assert_update_signup(api_client, signup_pk, signup_data, query_string=None):
 
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
-def test__registration_admin_can_update_signup(
+def test_registration_admin_can_update_signup(
     user_api_client, registration, signup, user
 ):
     user.get_default_organization().registration_admin_users.add(user)
 
-    new_signup_name = "Edited name"
-
     db_signup = SignUp.objects.get(pk=signup.id)
     assert db_signup.first_name != new_signup_name
     assert db_signup.last_modified_by_id is None
 
     signup_data = {
         "registration": registration.id,
         "first_name": new_signup_name,
-        "date_of_birth": "2015-01-01",
+        "date_of_birth": new_date_of_birth,
     }
 
     assert_update_signup(user_api_client, signup.id, signup_data)
-    db_signup = SignUp.objects.get(pk=signup.id)
+    db_signup.refresh_from_db()
     assert db_signup.first_name == new_signup_name
     assert db_signup.last_modified_by_id == user.id
 
 
 @freeze_time("2023-03-14 03:30:00+02:00")
 @pytest.mark.django_db
-def test__regular_admin_cannot_update_signup(user_api_client, registration, signup):
-    new_signup_name = "Edited name"
+def test_created_admin_can_update_signup(registration, signup, user, user_api_client):
+    signup.created_by = user
```

Comment:
Using `SignUpFactory` might be a better solution (and that could be used throughout the test case without fetching separately from DB).